Lib/ast.py

```diff
@@ -904,7 +952,9 @@ def _fstring_Constant(self, node, write):
 
     def _fstring_FormattedValue(self, node, write):
         write("{")
-        expr = type(self)().visit(node.value).rstrip("\n")
+        unparser = type(self)()
```

Comment:
Should this method be a classmethod? ` type(self)()` looks suspicious especially since we don't use `self` in the function anymore.